app/controllers/declaration/DestinationCountriesController.scala

```diff
@@ -69,14 +61,14 @@ class DestinationCountriesController @Inject()(
     hc: HeaderCarrier
   ): Future[Result] =
     customsCacheService.fetchAndGetEntry[DestinationCountriesSupplementary](cacheId, formId).map {
-      case Some(data) => Ok(destination_countries_supplementary(supplementaryForm.fill(data)))
-      case _          => Ok(destination_countries_supplementary(supplementaryForm))
+      case Some(data) => Ok(destination_countries_supplementary(supplementaryForm.fill(data), Countries.allCountries))
```

Comment:
What do you think to move import to view directly?

Reply:
Done